tests/regression_tests/surface_tally/test.py

```diff
@@ -106,19 +106,19 @@ def __init__(self, *args, **kwargs):
 
         # Create partial current tallies from water to fuel
         # Filters
-        cell_from_filter = openmc.CellFromFilter(water)
+        mat_from_filter = openmc.MaterialFromFilter(borated_water)
```

Comment:
The fact that the results didn't change here is good! We'll probably want to do this with two tallies though so we're still testing the `CellFromFilter` and that the surface currents are the same when using either a `CellFromFilter` or the `MaterialFromFilter`.

Reply:
there were 2 usage of the `openmc.CellFromFilter(water)`, so I figured I could divert 1, 
I can also implement an additional test.

Reply:
Hrmmm yeah that's probably okay. Ideally, we'd have dedicated test cases for the Cell/MaterialFromFilter's rather than wrap them into this surface tally test, but that shouldn't be on you to fix since it's a pre-existing issue. Would you be willing to make an issue so we can track that though? (in case one of us magically has time to address it in the future) Thanks!

Reply:
I'll do that !
thanks for the review